ambry-test-utils/src/main/java/com/github/ambry/clustermap/MockClusterMap.java

```diff
@@ -671,6 +671,16 @@ public ClusterMapChangeListener getClusterMapChangeListener() {
     return clusterMapChangeListener;
   }
 
+  /**
+   * removeDataNode to invoke clustermap change listener. This method doesn't really remove the datanode.
+   * @param dataNodeId The removed data node
+   */
+  public void removeDataNode(DataNodeId dataNodeId) {
```

Comment:
Can we update the method name to be something like invokeListenerWhenDataNodeRemoved since this method does not remove the data node?

Reply:
updated